src/components/CodeView/index.spec.tsx

```diff
@@ -138,6 +147,29 @@ describe(__filename, () => {
     return { root, selectedMessageMap };
   };
 
+  const renderSlowLoadingCode = ({
+    _slowLoadingLineCount = 3,
+    content,
+    contentLineCount,
+    ...moreProps
+  }: RenderParams & { contentLineCount?: number } = {}) => {
+    return renderWithLinterProvider({
+      _slowLoadingLineCount,
+      content:
+        content ||
```

Comment:
It's a bit hard to read because of Prettier, I think. Could you maybe extract this variable above the `return` call?

Alternatively, maybe the `Array().fill().join()` could be used as default value in the function declaration (`contentLineCount` would need to be list first/before `content`).

Reply:
Sure. I was thinking of putting it in the function signature but I didn't want to define the types (too lazy 😅 ).